feature-libs/asm/styles/components/_customer-selection.component.scss

```diff
@@ -63,27 +57,23 @@
     }
 
     button[type='submit'] {
-      @include forVersion(1, 5) {
```

Comment:
It looks like this block was kept, when it should have been dropped.  Could you please re-verify this change?

Reply:
Ah, you are right,  I am careless. now I have corrected it